EIPS/eip-7213.md

```diff
@@ -0,0 +1,255 @@
+---
+eip: 7213
```

Comment:
```suggestion
eip: [to be assigned]
```
EIP numbers are assigned by EIP editors/associates.  I can't find a record of this number being assigned.

Reply:
yes you are right @abcoathup , please could you assign one to this EIP?